web/src/views/LoginPortal/SecondFactor/MethodContainer.tsx

```diff
@@ -95,21 +97,22 @@ interface NotRegisteredContainerProps {
 }
 
 function NotRegisteredContainer(props: NotRegisteredContainerProps) {
+    const { t } = useTranslation("Portal");
```

Comment:
```suggestion
```

Assuming the other const gets moved up a level you should be able to remove this.